neqo-client/src/main.rs

```diff
@@ -241,6 +241,7 @@ fn client(args: Args, socket: UdpSocket, local_addr: SocketAddr, remote_addr: So
         Connection::new_client(
             args.url.host_str().unwrap(),
             &args.alpn,
+            FixedConnectionIdManager::make(0),
```

Comment:
I think the ownership semantics are better defined by the creator, rather than the createe (or at least, this is more idiomatic.) So we should rename `make` to `new` and then have the caller wrap with Rc::new(RefCell::new()) if that's what's called for.